### PR TITLE
Power armor is now actually powered

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

- Power armor requires power cells to work. When it runs out of charge - you lose the traits and a bit of speed.
- Salvaged PA is slower now, but a bit faster than unpowered PA.
- Low damage(<18) bullets bounce off the power armor now.
- Salvaged PA is no longer under the power armor pathing, it's now just armor.

## Why It's Good For The Game

- Not much just yet, but eventually we'll find more things to staple onto power armor that would consume the charge. Makes power armor, well, _powered_.
- I never understood why salvaged PA, literally a giant brick of metal, is faster than normal PA in some cases. This fixes it. Salvaged PA should be just a set of heavy armor and a really heavy one at that.
- Shrapnel shouldn't be a bane of giant brick of metal, I think.
- It's good for the code. Additionally, all helmet armor values correspond to the suit values.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/_onclick/item_attack.dm [hunk 1]
```diff
@@ -93,6 +93,7 @@
 	var/buffout = force*0.25
 	var/smutant = force*0.25
 	var/ghoulmelee = force*0.25 //negative trait, this will cut 25% of the damage done by melee
+	var/power_armor = force*0.25
```
Comment (chemistrymain2) [inline code/_onclick/item_attack.dm:96]: No, this combined with bigleagues and buffout would very nearly double the damage of any given melee weapon and make things like super sledges crit you in literally one hit

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -1406,6 +1406,9 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 			damage *= 0.8
 		//END OF CITADEL CHANGES
 
+		if(HAS_TRAIT(user, TRAIT_POWER_ARMOR)) // Being in power armor increases unarmed damage
+			damage *= 1.4
```
Comment (chemistrymain2) [inline code/modules/mob/living/carbon/human/species.dm:1410]: Combat mode + iron fist + power armor melee buff + mace gloves = instant crit.

Let's not.

File: code/modules/clothing/suits/f13armor.dm [hunk 3]
```diff
@@ -320,15 +408,21 @@
 		to_chat(L, "<span class='warning'>Armor power reroute successful. All systems operational.</span>")
 		L.update_equipment_speed_mods()
 
+/obj/item/clothing/suit/armor/f13/power_armor/run_block(mob/living/owner, atom/object, damage, attack_text, attack_type, armour_penetration, mob/attacker, def_zone, final_block_chance, list/block_return)
+	if(def_zone in protected_zones)
+		if(prob(70) && (damage < 18) && (armour_penetration < 0.15)) // Weak projectiles like shrapnel get deflected
```
[the remark this answers is not in the document]
Comment (5467543) [reply code/modules/clothing/suits/f13armor.dm:413]: ^

**Comment by 5467543** [comment]
FOURTY PERCENT damage increase from being in PA
FOURTY
PERCENT
do you recognize that anyone can just put on some sappers and 1v100 people with that level of damage

**Comment by EgorDinamit** [comment]
Removed melee buffs and cleaned up the salvaged PA bullshit, it's no longer under power armor path.

**Comment by EgorDinamit** [comment]
> how would the cell be changed? just use in hand, use tools, what? and will the charge be visible for the person wearing the armor?

You can remove cell by using the suit in hand.
Charge level is shown on examine message.

**Comment by ScratKnapp** [comment]
Does T-51 require charging with cells, seeing as the suit is fusion powered?

**Pull request merged**